tests/emails/offerer_validation_test.py

```diff
@@ -0,0 +1,200 @@
+import secrets
+from unittest.mock import patch
+
+from bs4 import BeautifulSoup
+
+from tests.conftest import clean_database, mocked_mail
+from tests.test_utils import create_user, create_offerer, create_user_offerer
+from tests.utils.mailing_test import get_mocked_response_status_200
+from utils.mailing import make_validation_confirmation_email, write_object_validation_email
+
+
+@clean_database
+def test_make_validation_confirmation_email_offerer_user_offerer_admin(app):
+    # Given
+    user = create_user(email='admin@letheatresas.com')
+    offerer = create_offerer(name='Le Théâtre SAS')
+    user_offerer = create_user_offerer(user, offerer, is_admin=True)
+    # When
+    with patch('utils.mailing.find_user_offerer_email', return_value='admin@letheatresas.com'):
+        email = make_validation_confirmation_email(user_offerer, offerer)
```

Comment:
validation + confirmation ==> redondance
```suggestion
        email = make_validation_email(user_offerer, offerer)
```

Reply:
Vu que prochainement, cette fonction va être splitée en deux, on la laisse comme ça.